bundles/org.openhab.binding.amazonechocontrol/src/main/java/org/openhab/binding/amazonechocontrol/internal/Connection.java

```diff
@@ -691,12 +694,14 @@ public HttpsURLConnection makeRequest(String verb, String url, @Nullable String
                         throw new HttpException(code,
                                 verb + " url '" + url + "' failed: " + connection.getResponseMessage());
                     }
-                    try {
-                        Thread.sleep(2000);
-                    } catch (InterruptedException e) {
-                        logger.warn("Unable to wait for next call to {}", url, e);
-                    }
+                    Thread.sleep(2000);
                 }
+            } catch (InterruptedException e) {
```

Comment:
Since you are now allowing the throwing of interrupts, you might as well group the interrupt related exceptions together.
```suggestion
            } catch (InterruptedException | InterruptedIOException e) {
```